### PR TITLE
set lon and lat as public and prefix stop point in deserialization

### PR DESCRIPTION
**Pull request by nberard**
requirements for https://github.com/CanalTP/osm-tools

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -540,7 +540,7 @@ pub fn write_stop_points_to_csv<P: AsRef<Path>>(
     for sp in stop_points {
         let stop_area_ids = get_stop_area_ids_for_stop_point(&sp.id, &stop_areas);
         let csv_row = vec![
-            sp.id.to_string(),
+            format!("StopPoint:{}", sp.id),
```
Comment (TeXitoi) [inline src/lib.rs]: Is this change needed? @prhod

File: src/lib.rs [hunk 2]
```diff
@@ -654,7 +654,7 @@ pub fn write_routes_to_csv<P: AsRef<Path>>(routes: Vec<Route>, output_dir: P) {
 
         //writing the route/stop csv
         for s in &r.ordered_stops_id {
-            let row = vec![r.id.to_string(), s.to_string()];
+            let row = vec![r.id.to_string(), format!("StopPoint:{}", s)];
```
Comment (TeXitoi) [inline src/lib.rs]: same here

**Comment by nlehuby** [comment]
For the sake of consistency, it would be better to apply these changes for all object types (and not only stops). 
It is rather incoherent to have `Line:Relation:2456` but `StopPoint:node:2485`.

**Comment by datanel** [comment]
hello

**Comment by nberard** [comment]
agreed, i talked about it with pascal and we decided to do it later, @prhod should i refacto all?

File: src/lib.rs (hunk 1 above)
Comment (prhod) [reply src/lib.rs]: The idea is to keep the actual output format, the "StopPoint" part was in the id of the StopPoint object. We thought writing it at serialization was better.

**Comment by nberard** [comment]
see #35
